test/e2e/run_e2e.sh

```diff
@@ -128,13 +128,14 @@ if [ "$create_cluster" = true ]; then
     if [ -f base_cluster.yaml ]; then
       ./copy-certificates.py base_cluster.yaml head_cluster.yaml
     else
-      aws-account-creator refresh-certificates --registry-file head_cluster.yaml --create-ca
+      aws-account-creator refresh-certificates --registry-file head_cluster.yaml --create-ca --provider zalando-eks
     fi
 
     # Update cluster
     echo "Updating cluster ${CLUSTER_ID}: ${API_SERVER_URL}"
 
     clm provision \
+        --provider=zalando-eks \
```

Comment:
This flag is only valid for the control loop. `clm provision` supports all known providers.